R/col_types.R

```diff
@@ -352,7 +348,6 @@ str.col_spec <- function(object, ..., indent.str = "") {
 
     # Append the current indentation string to the specs
     paste(indent.str, specs, collapse = "\n"),
-
```

Comment:
In contrast, there's no easily discernible motivation for this blank line, so I am willing to let it go.